recipe/meta.yaml

```
@@ -1,40 +1,50 @@
-{% set version = "1.3.0" %}
+{% set name = "segregation" %}
+{% set version = "2.5.2" %}
 
 package:
-  name: segregation
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/pysal/segregation/archive/v{{ version }}.tar.gz
-  sha256: 5a2e67c95183ef6324454bfe69a18737288df451a8ce8cccc1cec092064767e4
+  url: https://github.com/pysal/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 25b4011853c6f3562af4fdabcf507fbadee98f43da38dcd59734667c272369b7
 
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
```

https://github.com/pysal/segregation/blob/main/pyproject.toml#L30

```suggestion
  skip: true  # [py<39]
```